test/protobuf/decoder_test.exs

```diff
@@ -126,7 +126,7 @@ defmodule Protobuf.Decoder.Test do
       use Protobuf, from: Path.expand("../proto/mumble.proto", __DIR__)
     end
 
-    msg = MumbleProto.Authenticate.new(username: "bitwalker")
+    msg = MumbleProto.MumbleProto.Authenticate.new(username: "bitwalker")
```

Comment:
@bitwalker @DylanGriffith what do you both think of separating the `inject` feature from this new namespacing feature? I agree that the new namespacing feature can have very surpising side-effects, but for teams that use packages it can be really nice to have the elixir modules match the namespacing of the protobuf definitions.

If I can make them both work, but mutually exclusive do you think that would be workable for the users of this library?

I would like to make the default behavior nest package namespaces under the module that calls `use Protobuf`. `inject` would flatten all the namespaces and would only really be useful for defining a single message since it will force the current module to contain the `new`, `encode` and `decode` functions. The new naming option `use_only_protobuf_names` would expand the packange and message namespaces into elixir style module names and would not be nested under the module that calls `use Protobuf`.

Alternatively we could defer all of this namespacing business and trim this PR back to just handling mutiple files and using package namespaces. Then we can re-open the namespacing questions in separate issues.